omise-woocommerce.php

```diff
@@ -20,6 +20,20 @@ class Omise {
 	 */
 	public $version = '3.10';
 
+	/**
+	 * @since 3.11
+	 *
+	 * @var array
+	 */
+	public $payment_methods = array(
+		'Omise_Payment_Alipay',
+		'Omise_Payment_Billpayment_Tesco',
+		'Omise_Payment_Creditcard',
+		'Omise_Payment_Installment',
+		'Omise_Payment_Internetbanking',
+		'Omise_Payment_Truemoney'
+	);
+
```

Comment:
Note, I prefer this way instead of constant because it gives a flexibility on altering its payment's classname in the future if we make it extensible. As it's quite common in WordPress world. (i.e. WordPress style is kinda heavily using `apply_filter( blah )` to open for a 3rd-party to make a change as they desire without editing the core code).